CSS/quiz.css

```diff
@@ -0,0 +1,41 @@
+body{
+	background-color: var(--light-primary-color);
+}
+
+.quiz-container{
+	align-items: center;
+}
+
+.question-container{
+	border-radius: 15px;
```

Comment:
you could have used rem

Reply:
I want to keep radius of 15px only so using absolute unit.